yewprint/src/menu.rs

```diff
@@ -150,27 +149,52 @@ impl Component for MenuItem {
     }
 }
 
-pub struct MenuDivider {}
+pub struct MenuDivider {
+    props: MenuDividerProps,
+}
+
+#[derive(Clone, PartialEq, Properties)]
+pub struct MenuDividerProps {
+    #[prop_or_default]
+    pub title: Option<yew::virtual_dom::VNode>,
+}
 
 impl Component for MenuDivider {
     type Message = ();
-    type Properties = ();
+    type Properties = MenuDividerProps;
 
-    fn create(_props: Self::Properties, _link: ComponentLink<Self>) -> Self {
-        Self {}
+    fn create(props: Self::Properties, _link: ComponentLink<Self>) -> Self {
+        MenuDivider { props }
```

Comment:
You can use `Self` when the struct you try to instantiate is the struct where it is implemented
```suggestion
        Self { props }
```